<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiUartSend</class>
 <widget class="QWidget" name="MultiUartSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="multiText_1">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>21</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiText_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_1">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>98</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>138</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>178</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>218</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>258</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiHexBtn_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>298</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiSendBtn_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="listLoopTime">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="listLoopSendBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>列表循环发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="listLoopCountsText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>循环次数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="listLoopCounts">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="listLoopTimeText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>每条消息时间间隔( 毫秒 )</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
